1/debian-10/rootfs/opt/bitnami/scripts/mediawiki-env.sh

```diff
@@ -98,6 +100,8 @@ MEDIAWIKI_SMTP_USER="${MEDIAWIKI_SMTP_USER:-"${SMTP_USER:-}"}"
 export MEDIAWIKI_SMTP_USER="${MEDIAWIKI_SMTP_USER:-}" # only used during the first initialization
 MEDIAWIKI_SMTP_PASSWORD="${MEDIAWIKI_SMTP_PASSWORD:-"${SMTP_PASSWORD:-}"}"
 export MEDIAWIKI_SMTP_PASSWORD="${MEDIAWIKI_SMTP_PASSWORD:-}" # only used during the first initialization
+MEDIAWIKI_ENABLE_SMTP_AUTH="${MEDIAWIKI_ENABLE_SMTP_AUTH:-"${SMTP_ENABLE_AUTH:-}"}"
+export MEDIAWIKI_ENABLE_SMTP_AUTH="${MEDIAWIKI_ENABLE_SMTP_AUTH:-}" # only used during the first initialization
```

Comment:
Adding a default value:
```suggestion
export MEDIAWIKI_ENABLE_SMTP_AUTH="${MEDIAWIKI_ENABLE_SMTP_AUTH:-yes}" # only used during the first initialization
```


I believe `yes` should be a default value because it used to be the previous hardcoded value. If not, users wanting to enable SMTP auth after upgrading will be forced to declare the new envvar in their docker-compose/deployment.